tests/unit/setup-unit-test.ts

```diff
@@ -75,13 +77,38 @@ module.exports = async () => {
   );
 
   const clicastTemplateCommand = `cd ${vcastEnvPath} && ${clicastExecutablePath.trimEnd()} -l C template GNU_CPP11_X`;
-  const { stdout, stderr } = await promisifiedExec(clicastTemplateCommand);
-  if (stderr) {
-    console.log(stderr);
+
+  // Clicast execution
+  const { stdout: stdoutClicast, stderr: stderrClicast } =
+    await promisifiedExec(clicastTemplateCommand);
+  if (stderrClicast) {
+    console.log(stderrClicast);
     throw new Error(`Error when running ${clicastTemplateCommand}`);
   }
 
-  console.log(stdout);
+  console.log(stdoutClicast);
+
+  // Construct the path to the tool_version.txt file
+  const toolVersionPath = path.join(
```

Comment:
Shouldn't we be using your `getToolVersion` function here?

Reply:
Used `getToolVersion` and replaced it to `utils.ts` in 27ab614ad1b4ca582f56f93af2111503196cfd58.